src/components/GameAbout/GameAbout.stories.tsx

```diff
@@ -1,42 +1,60 @@
-      '\n' +
-      'Morbi consectetur, turpis quis iaculis elementum, justo eros semper magna, at venenatis ligula orci at arcu. Vivamus rutrum, nulla pharetra accumsan convallis, turpis ex malesuada erat, in viverra leo leo malesuada mauris. Interdum et malesuada fames ac ante ipsum primis in faucibus. Vestibulum vitae lobortis dui. Quisque eget posuere leo. Nam maximus elit sed urna venenatis varius. Maecenas vitae dignissim velit. Cras lacus mi, rutrum consectetur sapien at, efficitur eleifend tortor. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas.\n' +
-      '\n' +
-      'Etiam ac lectus nulla. Vivamus sit amet tellus vel augue commodo fringilla sed in purus. Suspendisse risus sem, feugiat blandit lacus a, suscipit ornare risus. Maecenas tincidunt quam et arcu auctor, vitae semper tellus sollicitudin. Pellentesque eleifend scelerisque sem, non interdum ex convallis et. Vivamus aliquet quis ante sed condimentum. Vivamus ligula libero, dignissim eget imperdiet egestas, sollicitudin tincidunt lorem. In nulla mauris, accumsan id euismod sagittis, blandit at risus.\n' +
-      '\n' +
-      'Praesent aliquet at orci et ullamcorper. Nullam sit amet viverra enim. Suspendisse aliquet, erat eget congue egestas, diam tellus egestas felis, vitae pharetra velit leo id lectus. Pellentesque commodo, enim volutpat consectetur egestas, sapien tellus tempor libero, vel tincidunt metus arcu eu libero. Duis nulla quam, ornare sed mi at, pretium sodales magna. Curabitur tortor ex, tristique viverra semper at, convallis ut risus. Nam id dictum dui. Duis porttitor felis sit amet tellus varius, vel eleifend nunc rhoncus. Aliquam volutpat arcu eu metus faucibus, in malesuada lacus porta. Maecenas efficitur nunc ipsum, varius sodales enim finibus ac. Vivamus faucibus in neque eget pellentesque. Donec quis lacus nisl. Donec vitae cursus lorem. Etiam aliquam quam bibendum, porta sem finibus, commodo ante.\n' +
-      '\n' +
-      'Pellentesque pellentesque enim quis mauris elementum sollicitudin. Sed pharetra ultrices posuere. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Suspendisse gravida laoreet mauris, quis hendrerit quam. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Morbi sit amet feugiat quam, quis feugiat justo.\n' +
-      '\n' +
-      'Duis tristique neque eu est tincidunt, ac sodales elit luctus. Praesent eget euismod leo. Phasellus in tortor aliquet, maximus felis sit amet, feugiat nisi. Nulla ornare porta rhoncus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Nulla mollis leo eu odio consectetur, a cursus nunc euismod. Sed porta leo lacus, sed tristique enim pharetra quis. Donec at nibh sit amet erat aliquet ullamcorper eu vitae dui.'
+export const WithGameName: Story = {
```

Comment:
I know this is a simple component and kind of overkill, but it may be useful to write a very simple storybook test here for expanding and collapsing the description to learn the storybook testing tools